lib/livebook.ex

```diff
@@ -230,9 +230,7 @@ defmodule Livebook do
       config :livebook, :dns_cluster_query, dns_cluster_query
     end
 
-    if agent_name = Livebook.Config.agent_name!("LIVEBOOK_AGENT_NAME") do
-      config :livebook, :agent_name, agent_name
-    end
+    config :livebook, :agent_name, Livebook.Config.agent_name!("LIVEBOOK_AGENT_NAME")
```

Comment:
I'd say this should still be a conditional. Instead, set the default value in config/config.exs. :)

Reply:
Ok. I fixed it